cfy_manager/components/validations.py

```diff
@@ -174,6 +175,31 @@ def _validate_inputs():
             )
 
 
+def _validate_host(host, input_name):
+    sock = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+    try:
+        sock.connect((host, INTERNAL_REST_PORT))
```

Comment:
this must of course be a port that is open, but i can't think of a good candidate right now